.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Bug Report
+about: Report a bug or unexpected behavior
+title: ""
+labels: bug
+assignees: ''
+
+---
+
+### Description
+A clear and concise description of the issue or bug. 
+
+### Expected vs Actual Behavior
+Include what you expected to happen and what actually happened (e.g., when compared with upstream MLIR).
+
+### Steps to Reproduce and a Minimal Working Example (MWE)
+1. Describe the steps to reproduce the issue (commands, etc.).
+2. If applicable, describe the environment (e.g., operating system, Python version, etc.).
+3. Please provide a **Minimal Working Example (MWE)** that reproduces the issue.  
+An MWE is the smallest possible piece of code that demonstrates the problem.  
```

Comment:
"An M" did my head in but I think it's correct.

Reply:
I don't think we benefit a lot from this acrtonym :)

Reply:
I would prefer without it, this sentence could just be `Please make your reproducing example as small as possible`